src/containers/admin-dashBoard/ManagerRoles.tsx

```diff
@@ -39,13 +39,26 @@ const AdminSission = () => {
     let newState = !deleteModel;
     setDeleteModel(newState);
   };
+  // useEffect(() => {
+  //   if (tabName === 'all') {
+  //     setData(rolemanagement);
+  //   } else {
+  //     setData(rolemanagement.filter((item: any) => item.role === tabName));
+  //   }
+  // }, [tabName]);
+
```

Comment:
Do you anticipate using this commented out code in the future?

Reply:
Solved.